src/tqec/circuit/operations/operation.py

```diff
@@ -230,6 +229,10 @@ def __init__(
         with the correct tags. Otherwise, you might need to manually tag the operation with the
         `cirq.VirtualTag` and the `STIM_TAG`.
 
+        Warning:
+            you should use `make_observable` to create instances of this class.
+            If you do not, read attentively the documentation below.
```

Comment:
Should be worded as "To understand how to use X, please refer to the documentation."